netkan/netkan/common.py

```diff
@@ -52,3 +70,89 @@ def download_stream_to_file(download_url: str, dest_file: IO[bytes]) -> None:
                       timeout=60) as req:
         for chunk in req.iter_content(chunk_size=8192):
             dest_file.write(chunk)
+
+
+class BaseMessageHandler:
```

Comment:
Let's put `BaseMessageHandler` and `QueueHandler` in their own file(s). They look big and complex enough to deserve that; `common.py` is a surprising place to find them.

Reply:
That's valid, I put them there being a common thing, but it's really more related to queue management.